dev/AppNotifications/AppNotificationContentBuilder/AppNotificationContentBuilder.idl

```diff
@@ -0,0 +1,138 @@
+﻿import "oaidl.idl";
+import "ocidl.idl";
+
+[contract(AppNotificationsContract, 1)]
+runtimeclass AppNotificationButton
+{
+    Constructors
+    // Initializes a new instance of the ToastButton class.
+    AppNotificationButton()
+
+    // Initializes a new instance of the ToastButton class.
+    AppNotificationButton(String, String)
+
+    // Gets or sets additional options relating to activation of the toast button.New in Creators Update
+    ActivationOptions { Get }
+
+    // Gets or sets what type of activation this button will use when clicked.Defaults to Foreground.
+    ActivationType { Get }
+
+    // Gets app - defined string of arguments that the app can later retrieve once it is activated when the user clicks the button.Required
+    Arguments { Get }
+
+    // Gets or sets an identifier used in telemetry to identify your category of action.This should be something like "Delete", "Reply", or "Archive".In the upcoming toast telemetry dashboard in Dev Center, you will be able to view how frequently your actions are being clicked.
```

Comment:
Was toast telemetry in dev center a feature that shipped? Assuming it never shipped. If it never shipped, HintActionId can be removed.